docs-guides/source/convert-pytorch-workflow.md

```diff
@@ -24,73 +30,74 @@ import torchvision
 torch_model = torchvision.models.mobilenet_v2(pretrained=True)
 # Set the model in evaluation mode.
 torch_model.eval()
+```
+
+```{admonition} Set the Model to Evaluation Mode
+
+To ensure that operations such as dropout are disabled, it's important to set the model to evaluation mode (_not_ training mode) before tracing. This setting also results in a more optimized version of the model for conversion.
```

Comment:
`To ensure that operations such as dropout are disabled` ---> 
`To ensure that training time operations such as dropout, batchnorm with moving average mean etc, are disabled`

Reply:
you can remove "not training mode"

Reply:
Revised as suggested